src/metabase/models/login_history.clj

```diff
@@ -30,7 +31,7 @@
                  :timezone (timezone-display-name timezone))
           (update :timestamp (fn [timestamp]
                                (if (and timestamp timezone)
-                                 (u.date/with-time-zone-same-instant timestamp timezone)
+                                 (t/zoned-date-time (u.date/with-time-zone-same-instant timestamp timezone) timezone)
                                  timestamp)))
```

Comment:
not related to the fix itself but this tweaks means we always get back a ZonedDateTime rather than OffsetDateTime (depending on the DB) which means we can generate a nicer human-friendly timestamp in the emails